src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -11,36 +14,86 @@ export default function PopupWithForm({ name, onSwitch }) {
           onClick={() => onSwitch("")}
         />
         {name === "signup" && <FormSignUp onSwitch={onSwitch} />}
-        {name === "signin" && <FormSignIn onSwitch={onSwitch} />}
-        {name === "success" && <FormSucess onSwitch={onSwitch}/>}
+        {name === "signin" && <FormSignIn onSwitch={onSwitch} closeModal={() => onSwitch("")}/>}
```

Comment:
**Might be better**


it's better to pass the content as `children` rather than make checks inside `PopupWithForm`